templates/go-gin/operation.tmpl

```diff
@@ -3,27 +3,24 @@
 package {{ index .GetMetadata "package" }}
 
 import "github.com/gin-gonic/gin"
+import "fmt"
 
-{{- $needs_fmt := false }}
 {{- $needs_regexp := false }}
 
 {{- range $i, $parameter := .Parameters }}
-{{- if ne $parameter.Schema.MinLength 0 }}{{ $needs_fmt = true }}{{- end }}
-{{- if ne $parameter.Schema.MaxLength 0 }}{{ $needs_fmt = true }}{{- end }}
 {{- if ne $parameter.Schema.Pattern "" }}{{ $needs_regexp = true }}{{- end }}
 {{- end }}
 
 {{- range $name, $requestBody := .RequestBody.Content }}
 {{- range $name, $property := $requestBody.Schema.Properties }}
-{{- if ne $property.MinLength 0 }}{{ $needs_fmt = true}}{{- end }}
-{{- if ne $property.MaxLength 0 }}{{ $needs_fmt = true}}{{- end }}
 {{- if ne $property.Pattern "" }}{{ $needs_regexp = true }}{{- end }}
 {{- end }}
 {{- end }}
 
-{{ if $needs_fmt -}} import "fmt" {{- end }}
 {{ if $needs_regexp -}} import "regexp" {{- end }}
 
+var _ = fmt.Sprint("")
```

Comment:
Why is this here?

Reply:
So that I can remove all the `$needs_fmt` stuff.